datafusion/functions-array/README.md

```diff
@@ -21,7 +21,7 @@
 
 [DataFusion][df] is an extensible query execution framework, written in Rust, that uses Apache Arrow as its in-memory format.
 
-This crate contains functions for working with arrays, such as `array_append` that work with
+This crate contains functions for working with arrays, such as `array_append` that works with
```

Comment:
I think the original was more correct as `functions` is plural

Reply:
Addressed